enable/testing.py

```diff
@@ -256,7 +256,9 @@ def mouse_move(self, interactor, x, y, window=None,
                                         alt_down=alt_down,
                                         control_down=control_down,
                                         shift_down=shift_down)
-        window.get_pointer_position.return_value = (x, y)
+        if isinstance(window, _MockWindow):
```

Comment:
If one passes a real window how does the `mouse_move` actually happens? When a real window get_pointer_position is called it will not have the right (x, y) value. In sort the call will not fail but the desired affect will probably not happen either.